avaframe/ana3AIMEC/ana3AIMECCfg.ini

```diff
@@ -73,6 +73,22 @@ unit =
 ##relTh = 0.75|0.9|0.8
 
 
+[PLOTS]
+# if extraPlots true additional analysis plots are created
+extraPlots = True
+# for one to one comparison of two result variables or derived quantities with option to distinguish scenarios using scenario
+# comparison result variables options: resTypeFieldMax (or Min or Mean), maxresTypeCrossMax, sRunout, deltaSXY, zRelease,
```

Comment:
We don't have a list what these parameters mean, right? At least not a full one. 
I think the new/missing parameters should be added to the ana3Aimec documentation including their source code names plus the new figures should also be added to the docs

If you don't want to do this with this PR, please add an issue, but start this soon then

Reply:
I added an issue #909